asv/benchmarks/setup/bench_model.py

```
@@ -78,6 +75,42 @@ def teardown(self, robot, num_worlds):
+        self.renderer = ViewerGL(headless=True)
+        self.renderer.set_model(self._model)
+
+        # Rendering one frame
+        self.renderer.begin_frame(0.0)
+        self.renderer.log_state(self._state)
+        self.renderer.end_frame()
+
+        wp.synchronize_device()
+        self.renderer.close()
+
+    def teardown(self, robot, num_worlds):
```

Ruff (0.14.0)

Unused method argument: `robot`

(ARG002)